### PR TITLE
PUB-1664 Clear Artefacts

### PR DESCRIPTION
**Pull request by junaidiqbalmoj**
### JIRA link

https://tools.hmcts.net/jira/browse/PUB-1664

### Change description

Add page to delete the publication for the selected court. Delete publication link will be visible when system admin tries to delete the court but there is a publication for that court.

image

image

image

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

File: src/main/resources/locales/en/delete-court-publication-confirmation.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (ChrisS1512) [inline src/main/resources/locales/en/delete-court-publication-confirmation.json:1]: I think this should be 'publications'. Same on line 5 (and will be the same in the other local files)

File: src/main/controllers/DeleteCourtSubscriptionConfirmationController.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            }
+            if (response === null) {
+                res.render(pageToLoad, {
+                    ...cloneDeep(req.i18n.getDataByLanguage(req.lng)[pageToLoad]),
+                    court: locationService.formatCourtValue(court),
+                    apiError: true,
+                    errorMessage:
+                        pageToLoad == 'delete-court-subscription-confirmation'
+                            ? 'Unknown error when attempting to delete all the subscription for the court'
+                            : 'Unknown error when attempting to delete all the artefact for the court',
```
Comment (ChrisS1512) [inline src/main/controllers/DeleteCourtSubscriptionConfirmationController.ts:46]: artefact should be artefacts here

File: src/main/views/delete-court-publication-success.njk [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+{% block beforeContent %}
+  {{ super() }}
+{% endblock %}
+{% block content %}
+  {{ successPanel(panelTitle, panelBodyMessage) }}
+  <p class="govuk-body govuk-!-font-weight-bold govuk-!-margin-top-6">{{ nextMessage }}</p>
+  <p class="govuk-body govuk-!-margin-0 govuk-!-font-weight-bold"><a class="govuk-link" href="delete-court-reference-data">{{ removeCourt }}</a></p>
```
Comment (ChrisS1512) [inline src/main/views/delete-court-publication-success.njk:13]: As discussed separately, this should be re-worded / re-direct the user back to the delete court page for the specific court

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

File: src/main/resources/locales/en/delete-court-publication-confirmation.json (hunk 1 above)
Comment (junaidiqbalmoj) [reply src/main/resources/locales/en/delete-court-publication-confirmation.json:1]: fixed.

File: src/main/controllers/DeleteCourtSubscriptionConfirmationController.ts (hunk 2 above)
Comment (junaidiqbalmoj) [reply src/main/controllers/DeleteCourtSubscriptionConfirmationController.ts]: fixed.

File: src/main/views/delete-court-publication-success.njk (hunk 3 above)
Comment (junaidiqbalmoj) [reply src/main/views/delete-court-publication-success.njk:14]: new link added.

**Review by ChrisS1512 — APPROVED** [review]

**Review by KianKwa — DISMISSED** [review]

**Review by KianKwa — DISMISSED** [review]

**Review by KianKwa — APPROVED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**